src/module-elasticsuite-indices/Model/IndexStatusProvider.php

```diff
@@ -112,7 +121,8 @@ public function __construct(
      */
     public function getIndexStatus($indexName, $alias): string
     {
-        $indexDate = $this->getIndexUpdatedDateFromIndexName($indexName, $alias);
+        $indexData = $this->indexSettings->parseIndexName($indexName);
+        $indexDate = $indexData['suffix'];
```

Comment:
You will now throw an exception everytime you have an "external" index on the cluster, won't you ?